platform/docs/docs/migration-guide.md

```diff
@@ -0,0 +1,492 @@
+---
+sidebar_position: 10
+sidebar_label: Migration Guide
+---
+
+# Migration Guide
+
+On this page, we will provide a guide to migrating from OHIF v2 to v3. Please note
+that this document is a work in progress and will be updated as we move forward.
+
+
+## Summary of Changes
+
+OHIF v3 is a complete re-architecture of the OHIF v2 to make it more modular and
+easier to maintain. The main differences are:
+
+- Extensions don't inject their modules into the viewer, they will make them available
+  to be used by the viewer.
+- To use the modules provided by the extensions, you need to write a [Modes](./platform/modes/index.md). Modes
+are configuration objects that will be used by the viewer to load the modules.
+- App configuration structure is different, mainly the `servers` is renamed to `dataSources`.
+- The viewer UI is completely re-written in tailwindcss for better maintainability.
+- cornerstone-core and cornerstone-tools are deprecated and OHIF v3 is using the new Cornerstone3D rendering library and tools
+- A new CLI tool to help you create extensions and modes.
+
+New significant additions that might be useful for you that weren't available in OHIF v2:
+- [OHIF CLI](./development/ohif-cli.md)
+- [New Rendering Engine and Toolings](https://www.cornerstonejs.org/)
+- [Modes](./platform/modes/index.md)
+- [Mode Gallery](https://ohif.org/modes)
+- [Layouts](./platform/extensions/modules/layout-template.md)
+- [Data Sources](./platform/extensions/modules/data-source.md)
+- [Hanging Protocols](./platform/services/data/HangingProtocolService.md)
+- [URL Params](./configuration/url.md)
+
+
+## Configuration
+
+OHIF v3 has a new configuration structure. The main difference is that the `servers` is renamed to `dataSources`
+and the configuration is now asynchronous.
+
+- `StudyPrefetcher` is not currently supported in OHIF v3.
```

Comment:
What about having a separate section of things that are not supported in v3 but are supported in v2? I know it might be a lot but maybe we can pick the most important ones

Reply:
I will do it later